### DISCUSSION TITLE
eBPF

### DISCUSSION THREAD

**Discussion by devOwlish** [shopokodu/community#22] [Теми для подкасту]
Було б дуже цікаво послухати про те, як eBPF працює, які дає можливості, і чи є розробка на ньому прерогативою C кодерів, чи можна використовувати Go, Python, etc. 

Цікаві eBPF-based проекти:
* [Cilium](https://cilium.io/) | [GitHub](https://github.com/cilium/cilium)
* [Hubble](https://github.com/cilium/hubble) + [Hubble UI](https://github.com/cilium/hubble-ui)
* [Caretta](https://github.com/groundcover-com/caretta)
* [bpftune](https://github.com/oracle-samples/bpftune)

Деякі доповіді:
* [Keynote: Hello eBPF! Goodbye Sidecars? - Liz Rice, Chief Open Source Officer, Isovalent ](https://www.youtube.com/watch?v=0JFd0W2CcMw&pp=ygUEZWJwZg%3D%3D)
* [Calling OPA from eBPF, Through WASM, in the Kernel? You've Gone Mad! - Nandor Kracser, Cisco](https://youtu.be/JSKNch6piyY)
* [The Cilium Story - Why We Created Cilium - Thomas Graf, Isovalen](https://www.youtube.com/watch?v=FucmPqU_pvw&pp=ygUGY2lsaXVt)

**Comment by andriigrynenko** [comment]
Кілька ключових розробників eBPF українці. Було б круто когось з них запросити.

**Comment by andriigrynenko** [comment]
https://www.youtube.com/watch?v=Wb_vD3XZYOA

**Comment by Dinisimys2018** [comment]
Ви ж самі скинули проекти де майже вся кодова база на Го написана ))) user space частина як я розумію
Ну а kernel space вже треба пилити на no-runtime мовах ( С Rust C++ )
Але взагальному нове для мене, дуже дякую, було би цікаво
